app/assets/stylesheets/group.scss

```diff
@@ -2,40 +2,37 @@ $white: #ffffff;
 
 .title {
   color: $white;
-  font-size: 5vw;
+  font-size: 4vw;
   font-weight: bold;
 }
 
-.groups {
+.indexs {
   text-align: center;
 }
 
-.group {
+.index {
   background-color: rgba(244, 162, 23, 0.66);
   -moz-border-radius: 15px;
   -webkit-border-radius: 15px;
   border-radius: 15px;
   margin: 20px;
-  max-width: 1000px;
+  max-width: 500px;
```

Comment:
Properties should be ordered background-color, -moz-border-radius, -webkit-border-radius, border-radius, margin, margin-left, margin-right, max-width, padding